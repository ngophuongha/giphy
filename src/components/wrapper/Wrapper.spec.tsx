import { render } from "../../../test";
import { Wrapper } from "./Wrapper.styled";

describe("Styled Wrapper", () => {
  it("should match snapshot", () => {
    const { container } = render(<Wrapper />);
    expect(container).toMatchSnapshot();
  });
});
