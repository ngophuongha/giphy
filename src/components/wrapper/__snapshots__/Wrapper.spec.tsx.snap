// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Styled Wrapper should match snapshot 1`] = `
.c0 {
  background-color: #fff;
  color: #000000;
  overflow-x: hidden;
}

<div>
  <div
    class="c0"
  />
</div>
`;
